test/unit/Breadcrumbs.test.js

```diff
@@ -81,7 +81,7 @@ describe('Breadcrumbs', () => {
         expect(component.find('Link')).to.have.length(2);
       });
 
-      it('should link back to the regular search results page', () => {
+      xit('should link back to the regular search results page', () => {
```

Comment:
Changed this test to pending, because we do not want this behavior anymore since the "regular search results page" is not really resolved yet.